docs/dapps/smart-contracts/deploy-a-smart-contract-on-avalanche-using-remix-and-core.md

```diff
@@ -19,29 +19,23 @@ DApps and the user experience.
 
 Today, we will deploy and test a smart contract on Avalanche using Remix and MetaMask.
 
-## Step 1: Setting up MetaMask
+## Step 1: Setting up Core
 
-Log in to MetaMask -&gt; Click the Network drop-down -&gt; Select Custom RPC
+If you don't already have a Core wallet, follow this 
+[guide](https://support.avax.network/en/articles/6100129-core-extension-how-do-i-create-a-new-wallet)
+to create a new wallet. 
 
-![MetaMask network dropdown](</img/image(60).png>)
+If you want to use the **Avalanche C-Chain**, it can be selected from the networks list.
 
-### Avalanche Mainnet Settings
+To switch to the **Fuji test network**, go to Settings, select Advanced, and then turn Testnet Mode on.
 
-- **Network Name**: Avalanche Mainnet C-Chain
```

Comment:
Are these settings preserved in a reference section somewhere? I think it is important we have it somewhere incase a MM user wants to add the network and test network to MM

Reply:
Hmm, I think I only kept Fuji in a reference section. But good point.